rocAL/rocAL_pybind/amd/rali/pipeline.py

```diff
@@ -190,6 +190,7 @@ def build(self):
                                 self._BoxEncoder = True
```

Comment:
the line above needs to be re-evaluated.  Don't recommend using outputs[2].prev.prev.prev..